### ISSUE TITLE
Is it possible to disable the 'double click' auto edit when I double click on preview screen?

### ISSUE THREAD

**Issue by bikiniweekend** [shd101wyy/markdown-preview-enhanced#1428]
Is it possible to disable the 'double click' auto edit when I double click on preview screen?
It is annoying when I want to copy a text.

**Comment by shd101wyy** [comment]
double click? Did you have the `markdown-preview-enhanced.automaticallyShowPreviewOfMarkdownBeingEdited` setting set to false?

**Comment by bikiniweekend** [comment]
Yes, I have it set to false.
I show you a gif with the double click auto edit on preview:
![markdown_doubleclick_autoedit](https://user-images.githubusercontent.com/25708827/86359906-ece55400-bc71-11ea-962f-fe8f3694717a.gif)

**Comment by shd101wyy** [comment]
Seems like you are using the default markdown preview, not the preview provided by MPE.  
Please right click at your markdown file, then choose `Markdown Preview Enhanced: Open preview on the side` command.

**Comment by bikiniweekend** [comment]
Yes, I have it set to false.
I show you a gif with the double click auto edit on preview:
![markdown_doubleclick_autoedit](https://user-images.githubusercontent.com/25708827/86360799-2d919d00-bc73-11ea-8628-c1e86959c8bc.gif)

**Comment by bikiniweekend** [comment]
> Seems like you are using the default markdown preview, not the preview provided by MPE.
> Please right click at your markdown file, then choose `Markdown Preview Enhanced: Open preview on the side` command.

Oh you are right, I thought it was a preview from your extension, but it is the default preview.
I used ctrl+shift+v to open this preview:
![Screenshot_1](https://user-images.githubusercontent.com/25708827/86361383-09828b80-bc74-11ea-8eae-7e9e47dc3655.jpg)

**Comment by shd101wyy** [comment]
Yes somehow the shortcut stopped working... I haven't found the solution.

**Comment by bikiniweekend** [comment]
> Yes somehow the shortcut stopped working... I haven't found the solution.

Yep.
Anyway I found the native option to disable the annoying thing.
Thanks for checking it!
![Screenshot_2](https://user-images.githubusercontent.com/25708827/86361874-bf4dda00-bc74-11ea-95de-9a96a51337eb.jpg)
